kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -313,129 +334,136 @@ define([
             };
         }
 
-        // LIFECYCLE API
-        function renderParameters() {
```

Comment:
I broke renderParameters into several functions as it was a bit long and complicated.

Reply:
It is broken into `findPathParams`, `createParamWidget`, and `renderParameterRow`.

Reply:
this is great! much more readable. you may have already planned on this but could you add a little documentation to these methods?

Reply:
what do you think about renaming createParamWidget be renamed to something like `createFilePathWidget` or `createFilePathInput`? to separate further from paramWidget cruft 

same thoughts for `renderParameterRow` and references there to `parameterRow`

Reply:
Sure, I can rename them

Reply:
thank you! this helps so much with readability